scripts/on-start.sh

```diff
@@ -126,21 +185,26 @@ function is_configured() {
 }
 function configure_instance() {
     log "INFO" "configuring instance..........."
-    local mysqlshell="mysqlsh -u${replication_user} -ppassword"
+    local mysqlshell="mysqlsh -u${replication_user} -p${MYSQL_ROOT_PASSWORD}"
     local mysql="mysql -u${replication_user} -ppassword"
     #todo check for is cofigured first ? and set is restart_required=...
+#    retry 120 mysqlsh -urepl -p${MYSQL_ROOT_PASSWORD} --sql -e "select @@gtid_mode"
     retry 120 ${mysqlshell} --sql -e "select @@gtid_mode;"
     gtid=($($mysqlshell --sql -e "select @@gtid_mode;"))
     echo "gtid -- ${gtid[@]}"
     echo "${gtid[1]}"
     if [[ "${gtid[1]}" == "ON" ]]; then
         log "info" "---------------$report_host is already_configured-------------"
         already_configured=1
+#        dropMetadataSchema
         return
     fi
+    #todo joining for the the fist time...
     #     wait_for_host_online
-    retry 30 ${mysqlshell} -e "dba.configureInstance('${replication_user}@${report_host}',{password:'password',interactive:false,restart:true});"
-
+    # todo read password from env?
+    retry 30 ${mysqlshell} -e "dba.configureInstance('${replication_user}@${report_host}',{password:'${MYSQL_ROOT_PASSWORD}',interactive:false,restart:true});"
+    wait $pid
```

Comment:
recheck ```wait $pid```

Reply:
it's used to prevent the creatation of new server before the previous one proccess is stopped.